misk-actions/src/main/kotlin/misk/web/interceptors/LogRequestResponse.kt

```diff
@@ -10,7 +10,20 @@ package misk.web.interceptors
  *
  * If arguments and responses may include sensitive information, it is expected that the toString()
  * methods of these objects will redact it.
+ *
+ * Rate limiting is used to sample the number of requests logged. The value specified is the
+ * buckets/sec allotted per service per pod. By default we set rate limiting for both successes
```

Comment:
I think `log events per sec` makes more sense to say?

Reply:
Yeah sure